portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var curSlide = 0; //by default, display first slide
+const totalSlides = 3; //total number of slides
```

Comment:
consider determining this value programatically via document. getElementsByClassName instead of hardcoding it